src/main/java/net/hypixel/nerdbot/cache/suggestion/Suggestion.java

```diff
@@ -0,0 +1,146 @@
+package net.hypixel.nerdbot.cache.suggestion;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageHistory;
+import net.dv8tion.jda.api.entities.MessageReaction;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.dv8tion.jda.api.managers.channel.concrete.ThreadChannelManager;
+import net.hypixel.nerdbot.NerdBotApp;
+import net.hypixel.nerdbot.bot.config.BotConfig;
+import net.hypixel.nerdbot.util.Util;
+
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+@Getter
+@Log4j2
+public class Suggestion {
+
+    private final ThreadChannel thread;
+    private final Optional<Message> firstMessage;
```

Comment:
Change the variable to type Message and create a method that returns the Optional